dotnet/src/Connectors/Connectors.HuggingFace.UnitTests/MultipleHttpMessageHandlerStub.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Do we need `HttpMessageHandlerStub` given we have this class?

Reply:
This class is duplicated for each UnitTest, I just moved it to InternalUtilities and removed it from the other projects.